### PR TITLE
52428 add header and footer

### PR DESCRIPTION
**Pull request by diemntn-0165**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52428)

## WHAT (optional)
- Add header and footer

## HOW
- add file _header.html.erb, _footer.html.erb, style scss

## Evidence (Screenshot or Video)
Screenshot from 2022-08-17 13-15-30
Screenshot from 2022-08-17 13-22-08

### CODE REVIEW

**Review by fonqhoanq — APPROVED** [review]

**Comment by diemntn-0165** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,14 +8,20 @@
 
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
     <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <link href="assets/bootstrap.min.css" rel="stylesheet">
+    <link href="assets/font-awesome.min.css" rel="stylesheet">
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb]: e require nó vào application.scss đi

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <li><a href="#"><%= t(".our_team")%></a> </li>
+                <li><a href="#"><%= t(".hiring")%></a> </li>
+            </ul>
+        </div>
+        <div class="col-xs-12 col-sm-2 how-it-works-links color-gray">
+            <h5><%= t(".how_work")%></h5>
+            <ul>
+                <li><a href="#"><%= t(".location")%></a> </li>
+                <li><a href="#"><%= t(".chooose_meal")%></a> </li>
+                <li><a href="#"><%= t(".Pay via credit card")%></a> </li>
```
Comment (ngocvt-0484) [inline app/views/layouts/_footer.html.erb]: tên i18n lạ thế này
viết liền nhá e

**Comment by diemntn-0165** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**